Decred Wallet/Features/Wallet Utils/SpendingPinOrPassword.swift

```diff
@@ -38,30 +41,26 @@ struct SpendingPinOrPassword {
         securityVC.securityFor = LocalizedStrings.spending
         securityVC.initialSecurityType = self.currentSecurityType()
         
-        securityVC.onUserEnteredPinOrPassword = { (newPinOrPassword, securityType) in
-            self.changeWalletSpendingPassphrase(vc, current: currentPinOrPassword, new: newPinOrPassword, type: securityType)
+        securityVC.onUserEnteredPinOrPassword = { (newPinOrPassword, securityType, securityRequestVC) in
+            self.changeWalletSpendingPassphrase(vc, current: currentPinOrPassword, new: newPinOrPassword, type: securityType, securityRequestVC: securityRequestVC)
         }
-        
-        vc.navigationController?.pushViewController(securityVC, animated: true)
+        vc.present(securityVC, animated: true, completion: nil)
     }
     
-    private static func changeWalletSpendingPassphrase(_ vc: UIViewController, current currentPassphrase: String, new newPassphrase: String, type securityType: String) {
-        let newSecurityType = securityType.lowercased()
-        let progressHud = Utils.showProgressHud(withText: String(format: LocalizedStrings.changingSpendingPINPass, newSecurityType))
```

Comment:
Please confirm if this means we can retire this string translation.

Reply:
I confirm it and what is more, I retired it.